sys/include/ztimer.h

```diff
@@ -301,13 +301,26 @@ struct ztimer_base {
     uint32_t offset;            /**< offset from last timer in list */
 };
 
+/**
+ * @defgroup    ztimer_now64 ztimer_now64
+ * @deprecated  use uint32_t or ztimer64, will be removed after 2022.07 release.
+ *
+ * @brief deprecated module changes deprecated ztimer_now_t type
```

Comment:
```suggestion
 * @brief 64-bit timestamp support
```

? Otherwise this will read very confusing in the overall documentation.

Reply:
I really wanted to make sure no one starts to use it (since this is adding documentation for it), maybe this is a bit to much.